docs/blog/2025-01-06-nx-update-20-3.md

```diff
@@ -213,7 +213,7 @@ We've also [added support for S3 compatible providers](/nx-api/powerpack-s3-cach
 
 We've also added the capability to allow folks to authenticate their Powerpack license via Nx Cloud (rather than requiring that the license be committed to the repository). This should enable oss library authors use Powerpack - without exposing their private license via their repo.
 
-We are offering free Nx Powerpack licenses to all open source libraries. We also offer free licenses to smaller businesses. If you are interested, you may [request a license now using Nx Cloud](https://cloud.nx.app/powerpack/purchase).
+We are offering free Nx Powerpack licenses to all open source libraries. We also offer free licenses to smaller businesses. If you are interested, you may [request a license now using Nx Cloud](/nx-enterprise/powerpack/free-licenses-and-trials).
```

Comment:
did you miss the UTM tag here?

Reply:
No it is still nx.dev, I changed the link altogether.